ruby/exercises/braille.rb

```diff
@@ -0,0 +1,63 @@
+require 'colorize'
+require_relative('../constants/brailleKey')
+exitPrompt = require_relative('../helpers/exitPrompt')
+quitToMenu = require_relative('../helpers/quitToMenu')
+
+def displayExitPrompt
+  exitPrompt
+end
+
+def separateLetters(array)
+  lettersArray = []
+  count = 1
+  index = 0
+
+  while(index < array.length)
+    if(count === 6)
+      letter = Array.new(array.slice(index - 5, 6))
+      lettersArray.push(letter)
+      count = 0
+    end
+    count += 1
+    index += 1
+  end
+
+  return lettersArray
+end
+
+def translateLetters(array)
+  translatedArray = []
```

Comment:
Whenever I see the pattern `declare temp collection, iterate over something and mutate temp collection, return temp collection`, I reach for a more functional approach:
```ruby
def translate( array ) 
  array.map { |letter| brailleKey.key( letter ).to_s }
end
```